src/config.cc

```diff
@@ -1157,7 +1157,8 @@ Class load()
 
   c.showingDictBarNames = ( root.namedItem( "showingDictBarNames" ).toElement().text() == "1" );
 
-  c.usingSmallIconsInToolbars = ( root.namedItem( "usingSmallIconsInToolbars" ).toElement().text() == "1" );
+  c.usingToolbarsIconSize =
+    ( static_cast< ToolbarsIconSize >( root.namedItem( "usingToolbarsIconSize" ).toElement().text().toInt() ) );
```

Comment:
as `usingToolbarsIconSize` does not exist on the first time when opening the gd-ng. what will happen?